packages/content/src/Application/SmartResolver/Resolver/SmartContentSmartResolver.php

```diff
@@ -65,6 +70,15 @@ public function resolve(SmartResolvable $resolvable, ?string $locale = null): Co
         $sortBys = $data['sortBys'] ?? [];
         $parameters = $data['parameters'];
 
+        if (isset($filters['audienceTargeting']) && $filters['audienceTargeting'] && $this->targetGroupStore) {
```

Comment:
Use the existing visitor pattern from the SmartContentPropertyResolver to add the filters. So that the bundles are decoupled (see e.g. mediabundle smart content filter visitor)

Reply:
This can be removed as there are already existing visitors